### ISSUE TITLE
Fix RSS Issue for Google News (All News)

### ISSUE THREAD

**Issue by slavakurilyak** [automatisch/automatisch#839]
automatisch v0.3.0 throws the following error when adding Google News RSS:

`internalId: must be string`

Here's the RSS feed for all news on Google News:

<img width="882" alt="1" src="https://user-images.githubusercontent.com/6625584/211680190-9a1511e3-dfb2-496b-b306-098c8a641183.png">

Here's all news on Google News:

https://news.google.com/home?hl=en-US&gl=US&ceid=US:en

<img width="882" alt="1" src="https://user-images.githubusercontent.com/6625584/211680349-aa9cc3c9-c4a4-42bf-8531-06e3a1a945c4.png">
<img width="1239" alt="2" src="https://user-images.githubusercontent.com/6625584/211680350-c16e28f8-cbc1-48bb-912a-a9d2a6ba82d5.png">

Here's the error log:

```
index.ts:71 Uncaught (in promise) Error: internalId: must be string
    at new t (index.ts:71:5)
    at QueryManager.ts:246:15
    at n (asyncMap.ts:30:30)
    at asyncMap.ts:19:47
    at new Promise (<anonymous>)
    at Object.then (asyncMap.ts:19:16)
    at Object.next (asyncMap.ts:31:39)
    at Xo (module.js:132:18)
    at Jo (module.js:176:3)
    at t.next (module.js:225:5)
t @ index.ts:71
(anonymous) @ QueryManager.ts:246
n @ asyncMap.ts:30
(anonymous) @ asyncMap.ts:19
then @ asyncMap.ts:19
(anonymous) @ asyncMap.ts:31
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
(anonymous) @ iteration.ts:13
uu @ iteration.ts:13
next @ Concast.ts:172
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
next @ index.ts:54
Xo @ module.js:132
Jo @ module.js:176
t.next @ module.js:225
(anonymous) @ createHttpLink.ts:164
Promise.catch (async)
(anonymous) @ useMutation.ts:107
(anonymous) @ index.tsx:86
We @ react-dom.production.min.js:52
Qe @ react-dom.production.min.js:52
(anonymous) @ react-dom.production.min.js:53
Ar @ react-dom.production.min.js:100
Fr @ react-dom.production.min.js:101
(anonymous) @ react-dom.production.min.js:113
Be @ react-dom.production.min.js:292
(anonymous) @ react-dom.production.min.js:50
Rr @ react-dom.production.min.js:105
Jt @ react-dom.production.min.js:75
Xt @ react-dom.production.min.js:74
t.unstable_runWithPriority @ scheduler.production.min.js:18
$o @ react-dom.production.min.js:122
je @ react-dom.production.min.js:292
Kt @ react-dom.production.min.js:73
```

**Comment by barinali** [comment]
@slavakurilyak, I expect this will be covered by #837 as well. I'll leave the mentioned issues in the pull request open until we publish a new release so that you can test them.

**Comment by barinali** [comment]
@slavakurilyak, we have just released version 0.4.0. Like other RSS issues, I expect this one to be resolved too. Could you please test this problem with the new version and let us know how it goes?

**Comment by slavakurilyak** [comment]
Issue resolved! 🕺
